install.sh

```diff
@@ -2,8 +2,16 @@
 
 umask 0022
 
-pushd scripts > /dev/null
-sudo -E ./install-hab.sh
-sudo -E ./hab-sup.service.sh
-sudo -E ./provision.sh
-popd > /dev/null
+type curl >/dev/null 2>&1 || { echo >&2 "curl is required for installation of habitat, but was not found. Exiting."; exit 1; }
+
+curl https://raw.githubusercontent.com/habitat-sh/on-prem-builder/master/NOTICE
+
+echo ""
```

Comment:
Totally trivial, but just `echo` by itself with no args will do what you want here.